app/stories/Pages.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {QueryRenderer} from 'react-relay';
+import {createMockEnvironment, MockPayloadGenerator} from 'relay-test-utils';
+import {withA11y} from '@storybook/addon-a11y';
+
+import Admin from 'pages/admin';
+// Import ApplicationReview from 'pages/application-review';
+// import Applications from 'pages/applications';
+import CiipApplicationSWRSImport from 'pages/ciip-application-swrs-import';
+// Import CiipApplication from 'pages/ciip-application';
+// import CompleteSubmit from 'pages/complete-submit';
+// import Index from 'pages/index';
+// import LoginRedirect from 'pages/login-redirect';
+// import OrganisationRequests from 'pages/organisation-requests';
+// import ProductsBenchmarks from 'pages/products-benchmarks';
```

Comment:
Are this imports here to remind us to test these pages?

Reply:
Yes they are (next PR)